internal/handlers/handlers.go

```diff
@@ -1 +1,79 @@
 package handlers
+
+import (
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+func HomeHandler(w http.ResponseWriter, r *http.Request) {
+	if r.URL.Path != "/" {
+		http.NotFound(w, r)
+		return
+	}
+
+	wd, err := os.Getwd()
+	if err != nil {
+		http.Error(w, "Internal server error", http.StatusInternalServerError)
+		return
+	}
+
+	possiblePaths := []string{
+		filepath.Join(wd, "index.html"),
+		filepath.Join(wd, "..", "index.html"),
+		filepath.Join(wd, "..", "..", "index.html"),
+	}
+
+	var indexPath string
+	for _, path := range possiblePaths {
+		if _, err := os.Stat(path); err == nil {
+			indexPath = path
+			break
+		}
+	}
+
+	if indexPath == "" {
+		http.Error(w, "index.html not found", http.StatusInternalServerError)
+		return
+	}
+
+	http.ServeFile(w, r, indexPath)
+}
+
+func UploadHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
+		http.Error(w, "Method not allowed", http.StatusMethodNotAllowed)
+		return
+	}
+
+	if err := r.ParseMultipartForm(10 << 20); err != nil {
+		http.Error(w, "File too large (max 10MB)", http.StatusBadRequest)
+		return
+	}
+
+	file, _, err := r.FormFile("myFile")
+	if err != nil {
+		http.Error(w, "Invalid file upload", http.StatusBadRequest)
+		return
+	}
+	defer file.Close()
+
+	content, err := io.ReadAll(file)
+	if err != nil {
+		http.Error(w, "Error reading file", http.StatusInternalServerError)
```

Comment:
Это ошибка клиента, что не отдал нам файл 
```suggestion
		http.Error(w, "Error reading file", http.StatusBadRequest)
```